libs/langchain/langchain/vectorstores/vectara.py

```diff
@@ -39,6 +39,7 @@ def __init__(
         vectara_corpus_id: Optional[str] = None,
         vectara_api_key: Optional[str] = None,
         vectara_api_timeout: int = 60,
+        source: Optional[str] = "langchain",
```

Comment:
late to the game here but quick nit, does this need to be Optional?

Reply:
Good catch. Thanks @baskaryan 
Should be: source: str = "langchain"
since it's merged - do you want to fix directly or should I submit a new PR?

Reply:
Up to you! If you put it up I can merge it in tonight, or I'll add it to my todos for tomorrow :)

Reply:
Fixing now.